cvat/apps/engine/task.py

```diff
@@ -551,14 +550,17 @@ def _update_status(msg: str) -> None:
     else:
         assert False, f"Unknown file storage {db_data.storage}"
 
+    if (
+        db_data.storage_method == models.StorageMethodChoice.FILE_SYSTEM and
+        not settings.MEDIA_CACHE_ALLOW_STATIC_CACHE
+    ):
+        db_data.storage_method = models.StorageMethodChoice.CACHE
```

Comment:
As we discussed, we doesn't make sense to provide such an option to users. As far as I remember, currently we don't provide any task creation logs on the requests page, where such a notification could be shown.

Reply:
> currently we don't provide any task creation logs on the requests page, where such a notification could be shown
 
 but we provide the current process status where such info is displayed
![image](https://github.com/user-attachments/assets/98a7606c-1667-45cd-afc8-88f27144ec81)

Reply:
So if there is such a log message, it will be immediately overwritten by the "Completed" status, won't it? We would generate this message in the very end of the task creation.